changelog/fragments/interactive-csv-cmd.yaml

```diff
@@ -0,0 +1,3 @@
+entries:
+    - description: Add interactive prompts while generating CSV.
```

Comment:
Usually, we do something as `Add flag --X to the command `operator-sdk xxx` in order to allow work with interactive prompts while generating CSV.